meson.build

```diff
@@ -953,7 +962,13 @@ bashcompletionslinks = {}
 subdir('include')
 subdir('lib')
 subdir('libblkid')
-subdir('libmount')
+
+if build_libmount
+  subdir('libmount')
+else
+  mount_dep = dependency('', required : false)
+  mount_static_dep = dependency('', required : false)
```

Comment:
Finally gave me the excuse to learn how Meson `disabler` works. That does seem like a much better way of handling dependencies between targets. I'll definitely make `libblkid` consistent with `libmount` using this pattern to keep them consistent.

I am wondering if it would still be beneficial or not to declare dependencies between _options_ using `require`. It seems like it would duplicate the logic all over the place, but might be easier to understand when a user tries to enable a tool to be built that requires `libmount` when they have disabled `libmount` from being built.

Reply:
Done. I've remove all of the additional option checks and `require` calls except for the one between `libblkid` and `libmount` which is necessary for the summary to look correct if `libblkid` is disabled. I added a similar check for `build-libfdisk` as `libfdisk` depends on `libblkid`. I can add the removed bits back in a separate commit if desired.